src/app/pages/home/services/tasks.service.ts

```diff
@@ -20,14 +20,24 @@ export class TasksService {
   }
 
   getTasksOfToday() {
```

Comment:
If this is only for Non Clinical tasks may be worth renaming it to reflect the same